### PR TITLE
Documentation migration

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,94 @@ maven-plugin
```
Comment (car-roll) [inline README.md]:
```suggestion
[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/maven-plugin)](https://plugins.jenkins.io/maven-plugin)
[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/maven-plugin?label=changelog)](https://github.com/jenkinsci/maven-plugin/blob/master/CHANGELOG.md)
[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/maven-plugin?color=blue)](https://plugins.jenkins.io/maven-plugin)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,454 @@
+## Release Notes
+
+#### Version 3.4 (July 31st, 2019)
+
+-   [Fix security
+    issue](https://jenkins.io/security/advisory/2019-07-31/#SECURITY-713)
+
+#### Version 3.3 (June 14th, 2019)
+
+-   ![(tick)](docs/images/check.svg){ Fixed: maven-plugin random socket leak, leading to
```
Comment (car-roll) [inline CHANGELOG.md]:
```suggestion
-   ![(tick)](docs/images/check.svg) Fixed: maven-plugin random socket leak, leading to
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,454 @@
+## Release Notes
+
+#### Version 3.4 (July 31st, 2019)
```
Comment (car-roll) [inline CHANGELOG.md]:
```suggestion
### Version 3.4
July 31st, 2019
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,454 @@
...
+
+-   [Fix security
+    issue](https://jenkins.io/security/advisory/2019-07-31/#SECURITY-713)
+
+#### Version 3.3 (June 14th, 2019)
+
+-   ![(tick)](docs/images/check.svg){ Fixed: maven-plugin random socket leak, leading to
+    threads leak on slave and master
+    ([JENKINS-57119](https://issues.jenkins-ci.org/browse/JENKINS-57119)
```
Comment (car-roll) [inline CHANGELOG.md]:
```suggestion
    ([JENKINS-57119](https://issues.jenkins-ci.org/browse/JENKINS-57119))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,454 @@
...
+-   ![(tick)](docs/images/check.svg) Fixed: [java.io](http://java.io).NotSerializableException:
+    The calling thread Thread has no associated
+    channel ([JENKINS-57244](https://issues.jenkins-ci.org/browse/JENKINS-57244)
```
Comment (car-roll) [inline CHANGELOG.md]:
```suggestion
    channel ([JENKINS-57244](https://issues.jenkins-ci.org/browse/JENKINS-57244))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,454 @@
...
+-   ![(tick)](docs/images/check.svg) Fixed: Upgrade maven embedder from 3.1.0 to at least
+    3.5.4 ([JENKINS-54530](https://issues.jenkins-ci.org/browse/JENKINS-54530)
```
Comment (car-roll) [inline CHANGELOG.md]:
```suggestion
    3.5.4 ([JENKINS-54530](https://issues.jenkins-ci.org/browse/JENKINS-54530))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -0,0 +1,454 @@
...
+
+-   ![(tick)](docs/images/check.svg) Fixed: remove anonymous classes
+    ([JENKINS-53481](https://issues.jenkins-ci.org/browse/JENKINS-53481)
+
+#### Version 3.1.2 (March 27th, 2018)
+
+-   ![(tick)](docs/images/check.svg) Fixed: JEP-200 failure to serialize Notifier, used
+    when \<ciManagement\> specifies email configuration. (  [
+    JENKINS-50251](https://issues.jenkins-ci.org/browse/JENKINS-50251) -
+    Getting issue details... STATUS  ) - Take 2 - fix a potential NPE
```
Comment (car-roll) [inline CHANGELOG.md]: WIll need to remove the jira status link text throughout the changelog: `Getting issue details... STATUS`

File: CHANGELOG.md (hunk 6 above)
Comment (car-roll) [reply CHANGELOG.md]: I think there are a few more unclosed parens scattered throughout

File: CHANGELOG.md [hunk 8]
```diff
@@ -0,0 +1,454 @@
...
+     Fixed: Allow to automatically trigger jobs when a
+    project import the dependencyManagement of another
+    ([JENKINS-15883](https://issues.jenkins-ci.org/browse/JENKINS-15883)
+-   ![(info)](docs/images/information.svg)
+    .emoticon-information} Internal: Update HttpClient and use the
+    client from [Apache HttpComponents Client 4.x API
+    Plugin](https://wiki.jenkins.io/display/JENKINS/Apache+HttpComponents+Client+4.x+API+Plugin) ([JENKINS-46053](https://issues.jenkins-ci.org/browse/JENKINS-46053),
+    [JENKINS-46210](https://issues.jenkins-ci.org/browse/JENKINS-46210)
+-   ![(info)](docs/images/information.svg)
+    .emoticon-information} Internal: Use the JSch library from the [JSch
```
Comment (car-roll) [inline CHANGELOG.md]:
```suggestion
    Internal: Use the JSch library from the [JSch
```
Couple of others lingering around

**Comment by olamy** [comment]
@oleg-nenashev what is the format to use for changelog?What is wrong here so we can fix it. It's a deprecated plugin (just security fixes) not sure Dependabot is something to really take care of for this plugin :)

File: pom.xml [hunk 9]
```diff
@@ -40,7 +40,7 @@ THE SOFTWARE.
 
   <name>Maven Integration plugin</name>
   <description>This plug-in provides, for better and for worse, a deep integration of Jenkins and Maven: Automatic triggers between projects depending on SNAPSHOTs, automated configuration of various Jenkins publishers (Junit, ...).</description>
-  <url>https://wiki.jenkins.io/display/JENKINS/Maven+Project+Plugin</url>
+  <url>http://github.com/jenkinsci/maven-plugin</url>
```
Comment (timja) [inline pom.xml]:
```suggestion
  <url>https://github.com/jenkinsci/maven-plugin</url>
```

File: README.md [hunk 10]
```diff
@@ -1,5 +1,98 @@
 maven-plugin
 ============
-
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/maven-plugin)](https://plugins.jenkins.io/maven-plugin)
+[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/maven-plugin?label=changelog)](https://github.com/jenkinsci/maven-plugin/blob/master/CHANGELOG.md)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/maven-plugin?color=blue)](https://plugins.jenkins.io/maven-plugin)
 Jenkins plugin for building Maven 2/3 jobs.
 See [Maven Project Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Maven+Project+Plugin) on Jenkins' Wiki for more information.
+
+Older versions of this plugin may not be safe to use. Please review the
```
Comment (timja) [inline README.md]: The security section shouldn't be here, the plugin site will automatically populate it from the update center API

**Comment by timja** [comment]
> @oleg-nenashev what is the format to use for changelog?What is wrong here so we can fix it. It's a deprecated plugin (just security fixes) not sure Dependabot is something to really take care of for this plugin :)

Is it worth calling out the deprecation? This is the most searched for plugin on google and second highest viewed plugin on the plugins site

File: README.md (hunk 10 above)
Comment (timja) [reply README.md]: @olamy this shouldn't be here, the information is duplicated and requires manual maintenance and syncing, the plugin site will handle it for you

**Pull request merged**